account_reversal/account_reversal.py

```diff
@@ -128,26 +124,14 @@ def create_reversals(self, cr, uid, ids, reversal_date,
 
         :return: Returns a list of ids of the created reversal moves
         """
-        if isinstance(ids, (int, long)):
-            ids = [ids]
-
-        reversed_move_ids = []
-        for src_move in self.browse(cr, uid, ids, context=context):
-            if src_move.reversal_id:
-                continue  # skip the reversal creation if already done
-
-            reversal_move_id = self._move_reversal(
-                cr, uid,
-                src_move,
+        return [
+            move._move_reversal(
                 reversal_date,
                 reversal_period_id=reversal_period_id,
                 reversal_journal_id=reversal_journal_id,
                 move_prefix=move_prefix,
-                move_line_prefix=move_line_prefix,
-                context=context
+                move_line_prefix=move_line_prefix
             )
-
-            if reversal_move_id:
-                reversed_move_ids.append(reversal_move_id)
-
-        return reversed_move_ids
+            for move in self
+            if move.reversal_id
```

Comment:
@bwrsandman can you just replace that by ```if not move.reversal_id```

Then it's all good for me, no need for another PR.